src/cogs/utilities.py

```diff
@@ -153,7 +158,7 @@ async def search(self, ctx: Context, *, query: str = ""):
         * `source`: The source of the sprite. This should be a sprite mod.
         * `modded`: Whether to only return modded tiles (either `true` or `false`).
         * `color`: The color of the sprite. This can be a color name (`red`) or a palette (`0/3`).
-        * `tiling`: The tiling type of the object. This must be one of `-1`, `0`, `1`, `2`, `3` or `4`.
+        * `tiling`: The tiling type of the object.
```

Comment:
Should the different tiling mode options be listed here?

Reply:
they should but i got lazy

Reply:
I can tell. Please fix this.